x-pack/plugins/enterprise_search/public/applications/workplace_search/views/role_mappings/role_mappings.tsx

```diff
@@ -14,16 +14,15 @@ import { EuiEmptyPrompt } from '@elastic/eui';
 import { FlashMessages } from '../../../shared/flash_messages';
 import { Loading } from '../../../shared/loading';
 import { AddRoleMappingButton, RoleMappingsTable } from '../../../shared/role_mapping';
-import { ViewContentHeader } from '../../components/shared/view_content_header';
-import { getRoleMappingPath, ROLE_MAPPING_NEW_PATH } from '../../routes';
-
 import {
   EMPTY_ROLE_MAPPINGS_TITLE,
-  EMPTY_ROLE_MAPPINGS_BODY,
-  ROLE_MAPPINGS_TABLE_HEADER,
   ROLE_MAPPINGS_TITLE,
   ROLE_MAPPINGS_DESCRIPTION,
-} from './constants';
+} from '../../../shared/role_mapping/constants';
+import { ViewContentHeader } from '../../components/shared/view_content_header';
+import { getRoleMappingPath, ROLE_MAPPING_NEW_PATH } from '../../routes';
+
+import { EMPTY_ROLE_MAPPINGS_BODY, ROLE_MAPPINGS_TABLE_HEADER } from './constants';
```

Comment:
I don't understand why the linter changed the order here but manually changing it back resulted in the linter putting it this way 🤷🏼‍♂️

Reply:
Makes sense to me FWIW, our import linter orders from farthest to nearest parent, so it wants `../../../` to come before `../../` and then orders alphabetically. Glad it was an easy fix though!